### ISSUE TITLE
Meeting notes

### ISSUE THREAD

**Issue by rpytel1** [rpytel1/efficient-semantic-segmentation#5]
# Meeting notes 2.05
## Problems
- What to do as a baseline solution?
- What semi-supervised methods can we use?
- How to add image priors?

## Baseline solution 
- We have UNet implemented
- Perhaps try different models (copy-paste Pytorch models)

## Semi-supervised methods
- Runaway from GANs at all cost.
- Cow-out and Cow-mix, Cutmix
- **Maybe extend later!!**

## Adding Image Priors
- main problems in image segmentation for AV is illumination changes, colour changes, occlusion changes.
- Occlusions: Cowmix/Cutmix
- illumination/colour changes: look for methods and filters for that.
-  Perhaps try [MISH](https://arxiv.org/pdf/1908.08681.pdf) -> better activation function

## Preliminary Milestones
1. Play with baseline (choose architecture) and do a literature study
2. Play with semi-supervised methods
3. Play with image priors (specialized filters etc) 
4. Report & celebrate

## Next week plan
1. Look for semi-supervised methods/specialized filters targeting image segmentation problems
2. Implement testing on validation set of Cityscapes => [Add testing on the competition testset (validation of Cityscapes)](https://github.com/rpytel1/efficient-semantic-segmentation/issues/2)
2. Try different architectures for a baseline setup. 
3. Try if adding basic transforms improves results.
4. Try if progressive resizing improves results.
